.zenodo.json

```diff
@@ -120,6 +139,11 @@
       "affiliation": "Deutsches Zentrum für Luft- und Raumfahrt e.V.",
       "type": "Other"
     },
+    {
+      "name": "Vyskočil, Jiří",
+      "affiliation":"CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0001-8822-0929"
```

Comment:
The type others is missing and is required any name in contributors: `"type": "Other"`

Reply:
Done